content/docs/user-guide/packaging/windows-release-builds.md

```diff
@@ -187,16 +189,16 @@ Your release build is now complete! Next, you need to [bundle your content](#bun
 
 The other option is to release your game plus the **O3DE Editor** and all of its tools, which is is a **non-monolithic** build. This is not recommended since the size of all these tools combined will be much larger than the **monolithic** build.
 
-A source engine supports non-monolithic projects by default. As detailed in [Prerequisites](#prerequisites), you should have already created a Visual Studio project for your project and registered your project to your engine.
+Building from a source engine supports non-monolithic projects by default. As detailed in [Prerequisites](#prerequisites), you should have already created a Visual Studio project for your project and registered your project to your engine.
 
-To build your non-monolithic project for release, use CMake to invoke Visual Studio builder. Specify the `INSTALL` target and `release` configuration.
+To build your non-monolithic project for release, use CMake to invoke Visual Studio builder. Specify the `install` target and `release` configuration.
```

Comment:
Actually the Visual Studio builder is "[MSBuild](https://learn.microsoft.com/en-us/visualstudio/msbuild/msbuild?view=vs-2022)" so we should use that term instead.

Reply:
Will do!